tests/parity_fabric/run_standalone_tasks.sh

```diff
@@ -39,5 +39,4 @@ retry_command() {
   return $exit_code
 }
 
-retry_command "python -m test_parity_ddp --accelerator="cpu" --devices=2 --tolerance=0.02"
```

Comment:
This is a test, it has to stay. Please don't remove this.

Reply:
That was asked by @carmocca, but happy to keep it...

Reply:
Where have I asked this?

Reply:
https://github.com/Lightning-AI/lightning/pull/17502#discussion_r1209399076

Reply:
ok, I was confused with the comment location...